File: sentimental-mario-less/mario.py
# print Mario's pyramid at the end of the stage using python
from cs50 import get_int

# While loop to check input is numerical and between 1 to 9
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break


for i in range(n):  # iterating for each row,
    for j in range(n - 1 - i):  # print " " for each row
        print(" ", end="")
    for j in range(i + 1):
        print("#", end="")  # print # for each row
    print()

File: mario-less/mario.c
// Prints Mario's pyramid at the end of the stage

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Size: "); // get the height of the pyramid ( by block)
    }
    while (n < 1);

    for (int i = 1; i <= n; i++) // determine the number of row
    {
        for (int j = 1; j <= n - i; j++)
        {
            printf(" "); // print the space, n - 1 will reduce the number of space as # increase
        }

        for (int j = 1; j <= i; j++) // loop make number # = n
        {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}
